fetch.bs

```diff
@@ -2644,6 +2644,11 @@ boolean <var>http3Only</var>, run these steps:
  <a for="connection timing info">secure connection start time</a> should be the result of calling
  <a for=/>unsafe shared current time</a> immmediately before starting the handshake process to
  secure <var>connection</var>. [[!TLS]]
+
+ <li><p>If <var>connection</var> is an HTTP/3 connection, <var>timingInfo</var>'s
+ <a for="connection timing info">connection start time</a> and <var>timingInfo</var>'s
+ <a for="connection timing info">secure connection start time</a> should be equal, as in HTTP/3
+ the handshake process is performed as part of the initial connection setup. [[!HTTP3]]
```

Comment:
It's not clear to me why this is a "should" and not a "must". I also separated the justification as we don't typically do those as part of a requirement.

We already expose the connection type through ALPN negotiated protocol so that's not a new concern.
```suggestion
 <li><p>If <var>connection</var> is an HTTP/3 connection, <var>timingInfo</var>'s
 <a for="connection timing info">connection start time</a> and <var>timingInfo</var>'s
 <a for="connection timing info">secure connection start time</a> must be equal. (In HTTP/3
 the secure transport handshake process is performed as part of the initial connection setup.)
 [[!HTTP3]]
```